src/modules/data/blocks/ticket/sagas.js

```diff
@@ -98,6 +123,7 @@ export function* setTicketInitialState( action ) {
 			put( actions.setTicketTempEndTime( clientId, endTime ) ),
 		] );
 	} catch ( err ) {
+		console.error( err );
```

Comment:
are we keeping this console errors or it's just with debugging purposes?

Reply:
Yes as right now we are catching all errors inside of those sections if an error happens the application will not report back the error to the client (browser) so we need to report back the error so QA or any user is able to share the error. 

Another thing on this one is a `console.error` that is going to be displayed only if `error` or `all` is enabled to display logs on client

Reply:
cc @juanfra